docs/writing-docs/doc-blocks.md

```diff
@@ -17,9 +21,39 @@ Storybook Docs automatically generates component props tables for components in
   />
 </video>
 
+This is extremely useful, but it can be further expanded. Additional information can be added to the component to better document it:
+
+```js
+// Button.js
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Button({isDisabled,content}) {
+  return (
+    <button disabled={isDisabled}>{content}</button>
+  );
+}
+
+Button.propTypes = {
+  /**
+   Checks if the button should be disable
+  */
+  isDisabled: PropTypes.bool.isRequired,
+  /**
+  The display content of the button
+  */
+  content: PropTypes.string.isRequired,
+};
+
+```
+
+By including the additional information, the args tables will be updated. Offering a richer experience for any stakeholders involved.
+
+
 #### DocsPage
 
-To use the ArgsTable in [DocsPage](./docspage#component-parameter), export a component property on your stories metadata:
+To use the ArgsTable in [DocsPage](./docs-page.md#component-parameter), export a component property on your stories metadata:
```

Comment:
I think if it is capitalised it should be in code ticks (ie either use "args table" [informal] or `ArgsTable` [referring to the doc block component]). Wdyt @jonniebigodes ?

Reply:
@tmeasday you're absolutely correct. I've adjusted accordingly.